### ISSUE TITLE
Catch-All properties

### ISSUE THREAD

**Issue by hoijui** [OPEN-NEXT/LOSH#54]
The two properties

```
okh:functionalMetadata
okh:productionMetadata
```

are kind of catch all things.
To me it is unclear how they should be used, and how one could make use of this data.
As I read their description, they want to store multiple structured values into a single unstructured field, for example:

```
omg:myFunnyMachine
  a okh:MOSH ;
  okh:functionalMetadata '''
  length: 70cm
  weight: 50kg
  ''' ;
  .
```

I think this is not what we want, as everyone could put different data in a different format in this field (someone else might put JSON data there, for example), and we already have a structured data format right there: RDF

I would think, we rather not have this data at all, then in this way.
Alternatively, this data could be there as separate fields, not from our standard, which - if widely adopted in the community - could be integrated into our standard. Until ten, it might look like this:

```
omg:myFunnyMachine
  a okh:MOSH ;
  schema:width '70cm' ;
  schema:weight '50kg' ;
  .
```

**Comment by moedn** [comment]
agree! that means that I'll define a set of standard metadata (for production this might be easy). However, when people start to use this as a custom field (which should be allowed since I don't see a way how to standardise functional metadata for such a huge variety of hardware) data should be displayed as well.

possibly related to #61
